packages/esm-patient-tests-app/src/test-results/dashboard.meta.ts

```diff
@@ -1,8 +1,8 @@
-export const dashboardMeta = {
+import { type DashboardConfig } from '@openmrs/esm-patient-common-lib';
+
+export const dashboardMeta: DashboardConfig = {
   slot: 'patient-chart-test-results-dashboard-slot',
-  columns: 1,
   path: 'Results',
   title: 'Results',
   icon: 'omrs-icon-chart-average',
-  hideDashboardTitle: true,
```

Comment:
Shouldn't we extend the `DashboardConfig` type to include `hideDashboardTitle` as an optional boolean property instead of removing them altogether. It looks like they're actively being used in [dashboard-view.component.tsx](https://github.com/openmrs/openmrs-esm-patient-chart/blob/main/packages/esm-patient-chart-app/src/patient-chart/chart-review/dashboard-view.component.tsx#L67).